### PR TITLE
Add support of ADAA PDF statements

### PR DESCRIPTION
**Pull request by kuchyale**
Added example of downloading bank account statements.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -40,12 +40,12 @@
             <dependency>
                 <groupId>cz.kb.openbanking.adaa.client</groupId>
                 <artifactId>api</artifactId>
-                <version>1.0.0</version>
+                <version>1.1.0-SNAPSHOT</version>
```
Comment (krtek) [inline pom.xml]: Let's make this `1.1.0` version. It's not a `SNAPSHOT` anymore. Is it?

File: pom.xml [hunk 2]
```diff
@@ -40,12 +40,12 @@
...
             </dependency>
             <dependency>
                 <groupId>cz.kb.openbanking.adaa.client</groupId>
                 <artifactId>jersey-impl</artifactId>
-                <version>1.0.0</version>
+                <version>1.1.0-SNAPSHOT</version>
```
Comment (krtek) [inline pom.xml]: Let's make this `1.1.0` version. It's not a `SNAPSHOT` anymore. Is it?

**Review by krtek — APPROVED** [review]

File: web/src/main/java/cz/kb/openbanking/adaa/example/web/resource/AccountStatementsResource.java [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    public Response getAccountStatements() {
+        // check access token
+        String accessToken = OAuth2FlowProvider.getAccessToken();
+        if (StringUtils.isBlank(accessToken)) {
+            return authorizationRedirect(uriInfo.getBaseUri());
+        }
+
+        Account account = new Account(Iban.valueOf(getIban()), Currency.getInstance(getCurrency()));
+        // get statements for last 30 days
+        OffsetDateTime fromDateTime = OffsetDateTime.now(ZoneId.systemDefault()).minusDays(30);
```
Comment (krtek) [inline web/src/main/java/cz/kb/openbanking/adaa/example/web/resource/AccountStatementsResource.java]: Maybe something like `minusDays(31).truncatedTo(ChronoUnit.DAYS)` to get full last 30 days.

File: pom.xml (hunk 1 above)
Comment (kuchyale) [reply pom.xml]: We need to add CI/CD tool (Jenkins or Travis CI) to build and release branches. Then after merge to master, we will release version 1.1.0

Comment (krtek) [reply pom.xml]: 👍

File: pom.xml (hunk 2 above)
Comment (kuchyale) [reply pom.xml]: After merge this PR, I'll release it to 1.1.0

**Pull request merged**